openmc/mesh.py

```diff
@@ -1527,8 +1518,24 @@ def volumes(self):
         return np.multiply.outer(np.outer(V_r, V_p), V_z)
 
     @property
-    def cartesian_vertices(self):
-        return self._convert_to_cartesian(self.vertices, self.origin)
+    def vertices(self):
```

Comment:
I think we need to issue a warning here and on `centroids` since it will probably break some peoples post-processing scripts

Reply:
Had the original version been part of a release I'd tend to agree, but given that the original version was also part of `0.13.4-dev` I'm not sure we need to?

Reply:
`vertices` and `centroids` have been available on `SphericalMesh` and `CylindricalMesh` since 0.13.1 through the `StructuredMesh` class. So they've been around awhile.

Reply:
Ah, I thought you were referring to `cartesian_vertices`. But even so the behavior of `vertices/coordinates` is changing so we should notify users about that. I'll update it soon.